src/main/java/org/kuali/kfs/fp/document/document/validation/DisbursementVoucherEmployeeInformationValidation.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ *
+ * Copyright 2005-2019 Kuali, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.kuali.kfs.fp.document.validation.impl;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.fp.businessobject.DisbursementVoucherPayeeDetail;
+import org.kuali.kfs.fp.document.DisbursementVoucherDocument;
+import org.kuali.kfs.kns.service.DataDictionaryService;
+import org.kuali.kfs.krad.util.GlobalVariables;
+import org.kuali.kfs.krad.util.MessageMap;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.KFSKeyConstants;
+import org.kuali.kfs.sys.KFSPropertyConstants;
+import org.kuali.kfs.sys.document.AccountingDocument;
+import org.kuali.kfs.sys.document.validation.GenericValidation;
+import org.kuali.kfs.sys.document.validation.event.AttributedDocumentEvent;
+import org.kuali.rice.kim.api.identity.Person;
+import org.kuali.rice.kim.api.identity.PersonService;
+
+public class DisbursementVoucherEmployeeInformationValidation extends GenericValidation {
+
+    private static final Logger LOG = LogManager.getLogger(DisbursementVoucherEmployeeInformationValidation.class);
+
+    public static final String DV_PAYEE_ID_NUMBER_PROPERTY_PATH = KFSPropertyConstants.DV_PAYEE_DETAIL + "." +
+            KFSPropertyConstants.DISB_VCHR_PAYEE_ID_NUMBER;
+
+    protected AccountingDocument accountingDocumentForValidation;
+    protected DataDictionaryService dataDictionaryService;
+    protected PersonService personService;
```

Comment:
It looks like this overlay was added just to increase the visibility of the service instance variables. Would it be helpful to add some comments or other documentation to make the intent of this overlay clearer, as well as whether it's pulling in a future KualiCo fix?

Reply:
Good point. I will add a comment and I have reported this to KualiCo.